ios/XMTPModule.swift

```diff
@@ -400,44 +400,31 @@ public class XMTPModule: Module {
 			}
 
 			let conversations = try await client.conversations.list()
-
-			return try await withThrowingTaskGroup(of: String.self) { group in
-				for conversation in conversations {
-					group.addTask {
-						await self.conversationsManager.set(conversation.cacheKey(inboxId), conversation)
-						return try ConversationWrapper.encode(conversation, client: client)
-					}
-				}
-
-				var results: [String] = []
-				for try await result in group {
-					results.append(result)
-				}
-
-				return results
+			
+			var results: [String] = []
```

Comment:
Went ahead and removed from listAll and listConversations since it could lead to similar issues with too many db connections, and I did not observe any slowdown in the test for loading 2000 conversations.

before the change I saw "`LOG  Loaded 1995 conversations in 5717ms`" and after I saw actually slightly faster "`LOG  Loaded 1995 conversations in 5595ms`"